cabot/cabotapp/models.py

```diff
@@ -160,7 +141,7 @@ class Meta:
         blank=True,
         help_text='Hipchat instance to send Hipchat alerts to (can be none if Hipchat alerts disabled).'
     )
-    hipchat_room_id = models.IntegerField(
+    hipchat_room_id = models.PositiveIntegerField(
```

Comment:
Good call, entirely forgot this would trigger a data migration. I just made a change to get the data migration to work and successfully ran it. I have included the schema migration file now.